java/src/main/java/org/galaxyproject/gxformat2/v19_09/Any.java

```diff
@@ -1,3 +1,17 @@
+// Copyright Common Workflow Language project contributors
```

Comment:
Think there's some inappropriate automatic code licensing going on, looks like license should be MIT
https://github.com/galaxyproject/gxformat2/blob/master/LICENSE unless I misunderstand what's going on